tkgi/manifest.json

```
@@ -4,14 +4,14 @@
   "manifest_version": "1.0.0",
   "name": "pivotal_pks",
   "creates_events": false,
-  "short_description": "Enterprise-Grade Kubernetes offering from Pivotal.",
+  "short_description": "Enterprise-Grade Kubernetes offering from VMware.",
   "guid": "b0090603-01c8-4ad9-8f9a-4f3700bf065b",
   "support": "core",
   "supported_os": [
     "linux",
     "mac_os"
   ],
-  "public_title": "Datadog-Pivotal Container Service Integration",
+  "public_title": "Datadog-Tanzu Kubernetes Grid Integrated Integration",
```

```suggestion
  "public_title": "Tanzu Kubernetes Grid Integrated Integration",
```
We can drop the Datadog- from titles